dotnetv3/CloudWatch/Scenarios/CloudWatchScenario.cs

```diff
@@ -0,0 +1,786 @@
+    private static async Task GetAndDisplayMetricStatistics(string metricNamespace, Metric metric)
+    {
+        Console.WriteLine(new string('-', 80));
+        Console.WriteLine($"3. Get CloudWatch metric statistics for the last day.");
+
+        for (int i = 0; i < _statTypes.Count; i++)
+        {
+            Console.WriteLine($"\t{i + 1}. {_statTypes[i]}");
+        }
+
+        var statisticChoiceNumber = 0;
+        while (statisticChoiceNumber < 1 || statisticChoiceNumber > _statTypes.Count)
+        {
+            Console.WriteLine(
+                "Select a metric statistic by entering a number from the preceding list:");
+            var choice = Console.ReadLine();
+            Int32.TryParse(choice, out statisticChoiceNumber);
+        }
+
+        var selectedStatistic = _statTypes[statisticChoiceNumber - 1];
+        var statisticsList = new List<string> { selectedStatistic };
+
+        var metricStatistics = await _cloudWatchWrapper.GetMetricStatistics(metricNamespace, metric.MetricName, statisticsList, metric.Dimensions, 1, 60);
+
+        if (!metricStatistics.Any())
+        {
+            Console.WriteLine($"No {selectedStatistic} statistics found for {metric} in namespace {metricNamespace}.");
+        }
+
+        metricStatistics = metricStatistics.OrderBy(s => s.Timestamp).ToList();
+        for (int i = 0; i < metricStatistics.Count && i < 10; i++)
+        {
+            var metricStat = metricStatistics[i];
+            var statValue = metricStat.GetType().GetProperty(selectedStatistic)!.GetValue(metricStat, null);
+            Console.WriteLine($"\t{i + 1}. Timestamp {metricStatistics[i].Timestamp:G} {selectedStatistic}: {statValue}");
+        }
+
+        Console.WriteLine(new string('-', 80));
+    }
+
+    /// <summary>
+    /// Get and display estimated billing statistics.
+    /// </summary>
+    /// <param name="metricNamespace">The namespace for metrics.</param>
+    /// <param name="metric">The CloudWatch metric.</param>
+    /// <returns>Async task.</returns>
+    private static async Task GetAndDisplayEstimatedBilling()
+    {
+        Console.WriteLine(new string('-', 80));
+        Console.WriteLine($"4. Get CloudWatch estimated billing for the last week.");
+
+        var billingStatistics = await SetupBillingStatistics();
+
+        for (int i = 0; i < billingStatistics.Count; i++)
+        {
+            Console.WriteLine($"\t{i + 1}. Timestamp {billingStatistics[i].Timestamp:G} : {billingStatistics[i].Maximum}");
+        }
+
+        Console.WriteLine(new string('-', 80));
+    }
+
+    // snippet-start:[CloudWatch.dotnetv3.GetMetricStatisticsSetup]
+    /// <summary>
+    /// Get billing statistics using a call to a wrapper class.
+    /// </summary>
+    /// <returns>A collection of billing statistics.</returns>
+    private static async Task<List<Datapoint>> SetupBillingStatistics()
+    {
+        // Make a request for EstimatedCharges with a period of one day for the past 7 days.
```

Comment:
spell our seven (because you need 10 or larger to use numerals).

Reply:
Fixed.